### ISSUE TITLE
How to cash out on the Teleport App

### ISSUE THREAD

**Issue by matt-brigante** [TeleportXYZ/TRIP-Guides#25]
### What's missing from the TRIP Guides

Would like to propose section on the left panel for the Teleport App and include this as a new guide. 

### Expected content

This will give users detailed instructions on how to cash out in the Teleport app by exporting their wallet to a recommended wallet provided. 

## Overview

In order to cash out your earnings and rewards, follow the steps provided below. 

Since Teleport uses Web3 technology to process each transaction, you will need download a web3 wallet and create a free account for a web3 exchange. This allows you to convert your USDC earnings to Cash in your bank account. You can learn more about web3 wallets [here](https://www.quicknode.com/guides/web3-fundamentals-security/basics-to-web3-wallets)  

## What you’ll need

Before we go into the step by step of cashing out, there are a couple things you’ll need to have ready before you can cash out. 

### 1. Ottr App (Or similar Solana iOS based Wallet)

Download the [[Ottr](https://apps.apple.com/us/app/ottr-wallet/id1628669270)](https://apps.apple.com/us/app/ottr-wallet/id1628669270) App, create a free account and wallet following the [[onboarding steps](https://help.ottr.finance/hc/en-us/articles/8847235037979-Getting-Started)](https://help.ottr.finance/hc/en-us/articles/8847235037979-Getting-Started).

We recommend using Ottr due to their simple and intuitive experience. Ottr also does not require SOL in your wallet in order to send USDC like most other wallets.* 

### 2. Coinbase App

Download the [[coinbase app](https://apps.apple.com/us/app/coinbase-buy-bitcoin-ether/id886427730)](https://apps.apple.com/us/app/coinbase-buy-bitcoin-ether/id886427730) (not the wallet) and [[create a free account](https://help.coinbase.com/en-au/coinbase/getting-started/getting-started-with-coinbase/create-a-coinbase-account)](https://help.coinbase.com/en-au/coinbase/getting-started/getting-started-with-coinbase/create-a-coinbase-account). Coinbase requires the following to create an account:

- Be at least 18 years old
- A [valid photo ID](https://help.coinbase.com/en-au/coinbase/getting-started/getting-started-with-coinbase/id-doc-verification)
- A passport, medicare card, or driver's licence
- A computer or smartphone connected to the internet
- A phone number connected to your smartphone (we'll send SMS text messages)
- The latest Coinbase app version

When prompted to add your bank account, you can do so now so you won’t have to add it when you go to cash out. 

Y*ou will use Coinbase to transfer your funds to your bank account after receiving them from Ottr (or your preferred Solana wallet). You can alternatively use any other exchange that supports SOL and USDC (Solana Network)*

### 1. Export your Teleport Wallet to Ottr (You will only have to do this one time)

The first step will require you to export your wallet to Ottr by copying your secret phrase (private key) either by writing it down on paper or copying it to your clipboard.

1. Open the Teleport App and tap **Profile** in the bottom menu bar.
2. Tap the green **Wallet** button in the top right of the screen.
3. An explanation of your secret phrase and privacy notice will pop up. Read this and then tap **Show Secret Phrase.**
4. You can now view your secret phrase, **copy to clipboard** or write it down. Make sure you do not share this with anyone as it will give them full control of your funds and wallet.
5. Once copied, you will now open the Ottr App (or Solana wallet app of your choice)
6. From the Ottr app, in the top middle you can see your current wallet. Tap this to open the **wallet menu**. (If you’re using another Solana wallet, find the import/add/connect wallet button usually associated with you wallet menu.)
7. Tap I**mport Wallet**. Enter your secret phrase and tap **Import Wallet**. Your earnings will now appear in the Ottr app. 

### 2. Send earnings from Ottr to Coinbase and cash out to a linked bank account

Next, you will send your USDC earnings to Coinbase to cash out.

1. Find your receiving Coinbase wallet address in the Coinbase app

- Click **Send & Receive**, click **Receive**
- Select **USD Coin** at the top
- Switch the **Network** to Solana
- Copy the USDC wallet address shown or scan the QR code

2. In the Ottr app, click into your USDC balance, click **Send**

3. Enter the receiving wallet address

4. Select USDC to send, enter the amount to send

5. Confirm send

6. In the coinbase app, you can now withdraw your USDC earnings to your linked Bank account by clicking **Cash Out**

**Comment by hwhelchel** [comment]
@matt-brigante from an information architecture perspective, why don't we add this under the Drivers section?

**Comment by abhilash710** [comment]
+1 on adding it to the Driver section. Don't have to call this out under the "Teleport" app or section

**Comment by hwhelchel** [comment]
@matt-brigante looks good to me to create a change request for this 👍 

Once the change request is created, myself and/or others may leave feedback on your specific implementation of this copy there in GitBook. Looking forward to seeing this.

**Comment by hwhelchel** [comment]
> The team is working quickly on bringing you an in-app cash out experience.

I wouldn't lead with something apologetic. It sets the frame like we're missing the mark. We don't need to voluntarily set that frame and feed that perception.

> Download the [[Ottr](https://apps.apple.com/us/app/ottr-wallet/id1628669270)](https://apps.apple.com/us/app/ottr-wallet/id1628669270) App, create a free account and wallet following the [[onboarding steps](https://help.ottr.finance/hc/en-us/articles/8847235037979-Getting-Started)](https://help.ottr.finance/hc/en-us/articles/8847235037979-Getting-Started).

I think a little more explanation of what a wallet is, what Ottr is, why we recommend it first before the steps would be good. Doesn't have to be paragraphs of text. Less is more. 

> (not the wallet)

If we're pointing them to the exact link to download, then I don't think we need this sidebar. Adds potential for mental friction and confusion. Maybe mention this later like "Note: If you have the Coinbase wallet app already this is a different app..." etc.

Basically a good rule of thumb is to just provide the standard normal happy path first as simply and as clearly as possible and then put relevant corner cases that users might run up against at the end of a section.

> (we’ll ask for proof)

Change grammatical person since we refers to Coinbase and not DEC or TRIP.

> Cash Out Steps

Again maybe a small explanation here of what wallet exporting is before going into the steps of how to do it.

**Comment by hwhelchel** [comment]
So it seems like the flow is:

`Teleport Wallet --> Ottr Wallet --> Coinbase Wallet`

Is it possible for users to simply do:

`Teleport Wallet --> Coinbase Wallet`

**Comment by hwhelchel** [comment]
Cool so a nice crisp explanation of that will go along way to helping people understand the why behind these steps. Again, no need to be apologetic or project a frame like we're missing the mark though with the current workflow here. 💪
